### ISSUE TITLE
WebApp - Re-check bug error page

### ISSUE THREAD

**Issue by ThaoNguyen998** [Koniverse/SubWallet-Extension#2779]
**Describe the bug**
Re-check bug error page

**To reproduce the bug**
Actual: Error page
![image](https://github.com/Koniverse/SubWallet-Extension/assets/130966515/e8308e68-d515-4438-ac45-dc077e847613)

![Image](https://github.com/Koniverse/SubWallet-Extension/assets/117999489/a1e23032-d57f-41cd-9ee7-a5afb21c4605)

![Image](https://github.com/Koniverse/SubWallet-Extension/assets/117999489/40d58a98-e717-4c58-8ae8-a41ff2f7b738)

**Comment by Thiendekaco** [comment]
After users clicked on any of the article navigation links, an issue occurred. Upon investigation, here's what I've found:

1. The user navigates to the page.
2. Meanwhile, a new version of the documentation is deployed (we've had numerous deployments in recent weeks).
3. The user attempts to open a new article by clicking a link.
4. The link fails to work because it references a JavaScript chunk that the user can no longer download. The name of the old chunk file has been changed, so users cannot retrieve the old file because it has been deleted.
5. Consequently, an error is thrown.

There are two potential solutions to address this issue:

1. Adjust the CDN settings to retain files from previous deployments for a certain period. For example, we can keep the previous chunks on our CDN for a week and automatically refresh the app for users if the bundle is more than a week old. This approach ensures that we don't need to worry about very old versions.

2. Implement client-side error handling ( including the use of react.lazy to catch errors and prompt the user to reload.)
```
window.addEventListener('error', e => {
  if (/Loading chunk [\d]+ failed/.test(e.message)) {
    window.location.reload();
  }
});
```
